priv/repo/migrations/20170711183723_add_admin_guest_active_fields_to_users.exs

```diff
@@ -0,0 +1,10 @@
+defmodule Agoneum.Repo.Migrations.AddAdminGuestActiveFieldsToUsers do
+  use Ecto.Migration
+
+  def change do
+    alter table(:account_users) do
+      add :active, :boolean, default: false
```

Comment:
Technically, Postgress allows for nulls here so we may want to add a not null constraint right?